app/models/trucking_audit.rb

```diff
@@ -13,4 +13,22 @@ def set_user_data
       self.company_id = user&.company_id
     end
   end
+
+  scope :alphabetical_sort, lambda {
```

Comment:
```suggestion
  scope :alphabetical_sort, -> {
```
используйте короткую запись